src/python/grpcio/grpc/_cython/cygrpc.pyx

```diff
@@ -45,6 +48,10 @@ include "grpc/_cython/_cygrpc/server.pyx.pxi"
 cdef class _ModuleState:
 
   def __cinit__(self):
+    filename = pkg_resources.resource_filename(__name__, '_windows/grpc.def')
```

Comment:
The Def file is irrelevant. However if you do know the absolute location of the dll, that simplifies things.